bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/local/TelldusLocalDeviceController.java

```diff
@@ -0,0 +1,287 @@
+import org.openhab.binding.tellstick.internal.local.json.TellstickLocalDevice;
+import org.openhab.core.library.types.IncreaseDecreaseType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.tellstick.JNA;
+import org.tellstick.device.TellstickDevice;
+import org.tellstick.device.TellstickDeviceEvent;
+import org.tellstick.device.TellstickException;
+import org.tellstick.device.TellstickSensorEvent;
+import org.tellstick.device.iface.Device;
+import org.tellstick.device.iface.DeviceChangeListener;
+import org.tellstick.device.iface.SensorListener;
+import org.tellstick.device.iface.SwitchableDevice;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+/**
+ * {@link TelldusLocalDeviceController} handles the communication with Telldus Local API (Tellstick ZNET v1/v2)
+ * This controller uses JSON based Rest API to communicate with Telldus Local API.
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+public class TelldusLocalDeviceController implements DeviceChangeListener, SensorListener, TelldusDeviceController {
+    private final Logger logger = LoggerFactory.getLogger(TelldusLocalDeviceController.class);
+    private long lastSend = 0;
+    public static final long DEFAULT_INTERVAL_BETWEEN_SEND = 250;
+    static final int REQUEST_TIMEOUT_MS = 5000;
+    private final HttpClient httpClient;
+    private final Gson gson = new Gson();
+    private String localApiUrl;
+    private String authorizationHeader = "Bearer ";
+    static final String HTTP_LOCAL_API = "api/";
+    static final String HTTP_LOCAL_API_DEVICES = HTTP_LOCAL_API + "devices/list?supportedMethods=19&includeIgnored=0";
+    static final String HTTP_LOCAL_API_SENSORS = HTTP_LOCAL_API
+            + "sensors/list?includeValues=1&includeScale=1&includeUnit=1&includeIgnored=0";
+    static final String HTTP_LOCAL_API_SENSOR_INFO = HTTP_LOCAL_API + "sensor/info";
+    static final String HTTP_LOCAL_API_DEVICE_DIM = HTTP_LOCAL_API + "device/dim?id=%d&level=%d";
+    static final String HTTP_LOCAL_API_DEVICE_TURNOFF = HTTP_LOCAL_API + "device/turnOff?id=%d";
+    static final String HTTP_LOCAL_DEVICE_TURNON = HTTP_LOCAL_API + "device/turnOn?id=%d";
+    private static final int MAX_RETRIES = 3;
+
+    public TelldusLocalDeviceController(TelldusLocalConfiguration configuration, HttpClient httpClient) {
+        this.httpClient = httpClient;
+        localApiUrl = "http://" + configuration.ipAddress + "/";
+        authorizationHeader = authorizationHeader + configuration.accessToken;
+    }
+
+    @Override
+    public void dispose() {
+    }
+
+    @Override
+    public void handleSendEvent(Device device, int resendCount, boolean isdimmer, Command command)
+            throws TellstickException {
+        logger.info("Send {} to {}", command, device);
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be debug.

Reply:
Fixed!